Src/IronPython/Runtime/Exceptions/StopIterationException.cs

```diff
@@ -61,10 +56,25 @@ public StopIterationException(string message, Exception innerException)
             get { return _traceback; }
             set { _traceback = value; }
         }
-    }
 
+        /// <summary>
+        /// Result of raise StopError(n)
+        /// </summary>
+        /// <returns>
+        /// value passed to StopError, or null if none
+        /// </returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1007:UseGenericsWhereAppropriate")]
+        [PythonHidden]
+        public object GetValue() {
```

Comment:
What about?
`public object GetValue() => (((IPythonAwareException)this).PythonException as PythonExceptions._StopIteration)?.value;`